src/Model/Table/QueueProcessesTable.php

```diff
@@ -286,15 +286,20 @@ public function endProcess(string $pid): void {
 	 * We might need to run some exec() kill command here instead.
 	 *
 	 * @param string $pid
-	 * @param int $sig Signal (defaults to graceful SIGTERM = 15)
+	 * @param int|null $sig Signal (defaults to graceful SIGTERM = 15)
 	 *
 	 * @return void
 	 */
-	public function terminateProcess(string $pid, int $sig = SIGTERM): void {
+	public function terminateProcess(string $pid, ?int $sig = 0): void {
```

Comment:
you need to revert signature, nullable is not an option now anymore.
mind the CI results, needs to be green